<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>slm_machineClass</class>
 <widget class="QMainWindow" name="slm_machineClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>224</width>
    <height>400</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Sans Serif</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>slm_machine</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icons/SLM_Logo</normaloff>:/icons/SLM_Logo</iconset>
  </property>
  <property name="locale">
   <locale language="Turkish" country="Turkey"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QListView" name="buddyList"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="fileTransferLabel">
        <property name="text">
         <string>File Transfer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="fileTransferProgressBar">
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="addBuddyButton">
        <property name="text">
         <string>Add Buddy</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="removeBuddyButton">
        <property name="text">
         <string>Remove Buddy</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>224</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionEncryption_Key"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_Qt"/>
    <addaction name="actionAbout_SLM"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="actionExit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icons/exit</normaloff>:/icons/exit</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/qtLogo</normaloff>:/icons/qtLogo</iconset>
   </property>
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionAbout_SLM">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/SLM_Alt_Logo</normaloff>:/icons/SLM_Alt_Logo</iconset>
   </property>
   <property name="text">
    <string>About SLM</string>
   </property>
  </action>
  <action name="actionEncryption_Key">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/encryption</normaloff>:/icons/encryption</iconset>
   </property>
   <property name="text">
    <string>Encryption Key</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
